app/controllers/api/v1/recipes_controller.rb

```diff
@@ -13,4 +13,18 @@ def index
     end
       render json: @recipes, :include => [:user]
   end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    @recipe.save
```

Comment:
Once we're authenticated, you could call `@recipe.user = current_user`, as `current_user` will be set as part of the authentication stuff. You'll want to do that before you save, and this will be preferable to/more secure than allowing the client to provide the user_id.